src/library-authoring/library-access/UserAccessWidget.jsx

```diff
@@ -167,19 +174,21 @@ UserAccessContainerBase.propTypes = {
   removeUserAccess: PropTypes.func.isRequired,
 };
 
-UserAccessContainer.propTypes = {
-  ...UserAccessContainerBase.propTypes,
+UserAccessWidget.propTypes = {
+  ...UserAccessWidgetContainerBase.propTypes,
   isUser: PropTypes.bool.isRequired,
   showRemoveModal: PropTypes.bool.isRequired,
   setShowRemoveModal: PropTypes.func.isRequired,
   showDeprivModal: PropTypes.bool.isRequired,
   setShowDeprivModal: PropTypes.func.isRequired,
 };
 
+export const RawUserAccessWidgetContainer = UserAccessWidgetContainerBase;
```

Comment:
I've changed this so that UserAccessWidgetContainerBase is exported as a constant instead.

I do want to preserve the 'container' suffix since we're using the [container pattern](https://www.freecodecamp.org/news/react-superpowers-container-pattern-20d664bdae65/) and want to make it clear what someone's importing. So I've improved the naming and exports to better reflect that.

Reply:
That's fair. A few places in the codebase still refers `UserAccessContainer` - consider updating them to be `UserAccessWidgetContainer` too for consistency.

Reply:
That's fair. A few places in the codebase still refers `UserAccessContainer` - consider updating them to be `UserAccessWidgetContainer` too for consistency.

Reply:
Done.